### DISCUSSION TITLE
创建网站配置问题

### DISCUSSION THREAD

**Discussion by zhengjianyang** [1Panel-dev/1Panel#6163] [Q&A]
### 1Panel 版本

v1.10.13-lts

### 问题描述

创建静态网站的时候会显示创建基于运行环境站点的配置

### 重现步骤

现创建一个运行环境的站点再创建静态站点就会出现运行环境的一些配置、刷新页面就正常了。

**Comment by zhengjianyang** [comment]
<img width="662" alt="image" src="https://github.com/user-attachments/assets/c05352f0-292c-4ed7-9e77-f504838ceca9">

**Comment by wanghe-fit2cloud** [comment]
感谢反馈，麻烦提供一下1Panel服务器配置、操作系统和使用浏览器等具体信息。
> 在本地环境未能复现该问题。
